### PR TITLE
Added multiple element preview on hover of scss selector

### PR DESCRIPTION
**Pull request by GauravB159**
Partial fix for #249

Still need to figure out the logic for specificity. There seems to be some kind of bug in the production build itself. 

Still need to add tests for the new feature.

### CODE REVIEW

**Comment by GauravB159** [comment]
image

Such cases have not been handled yet, only the case where an element's immediate parents are multiple has been handled. I'm not sure if we want to handle this case, since it might just get too complicated. If needed, I can possibly extend it to cover this as well using a recursive method.

**Comment by aeschli** [comment]
Yes, I agree that this can get very complicated. The hover size should not get too long. Maybe write (1 more) behind `<element class="div-two">` ?

**Comment by GauravB159** [comment]
Hmm, I'm not sure how to limit the depth. Possibly depends on what the most frequent use case for this would be. I don't think people would want to preview a really deep node list anyways. 

I'm not sure what you mean by write one more behind `<element class="div-two">`?

Basically, the way it currently works is, if the second to last nested selector has multiple values, it picks up those, but other than that it picks the first selector in each ancestor. 

Basically a depth of 1, I guess.

**Comment by aeschli** [comment]
Closing the PR due to inactivity. Happy to reopen if you are interested to continue.
